src/components/AlphabetFilter/AlphabetFilter.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { forwardRef, useRef, useState } from "react";
+import { Box, chakra, useMultiStyleConfig, VStack } from "@chakra-ui/react";
+import { Button } from "../Button/Button";
+import { Heading } from "../Heading/Heading";
+import { Text } from "../Text/Text";
+import useNYPLBreakpoints from "../../hooks/useNYPLBreakpoints";
+
+export interface AlphabetFilterProps {
+  /** Array of letters to specify which `Button` components should be set in an `enabled`
+   * state. By default, all buttons are `enabled`. */
+  activeLetters?: string[];
+  /** A class name for the AlphabetFilter parent div. */
+  className?: string;
+  /** The currentLetter can be used to programatically set the selected letter without the
+   * user explicitly requesting it. */
+  currentLetter?: string;
+  /** Value used to set the text for a `Text` component that will be displayed above the letter buttons. */
+  descriptionText?: string | JSX.Element;
+  /** Value used to set the text for a `Heading` component. */
+  headingText?: string;
+  /** ID that other components can cross reference for accessibility purposes. */
+  id?: string;
+  /** Adds the `disabled` prop to the AlphabetFilter when true. */
+  isDisabled?: boolean;
+  /** The callback function called when a letter button or the Show All button is clicked. */
+  onClick: (filterValue: string) => void;
+}
+
+export const AlphabetFilter = chakra(
+  forwardRef<HTMLDivElement, AlphabetFilterProps>((props, ref?) => {
```

Comment:
This is fine as it is setup, but the `ref` won't always be useful when we try to get the value from it. This is because the `ref` is being passed to the wrapper div, but the wrapper div doesn't capture any of the data that gets updated. There are also too many buttons and those can't use a single ref. This is not necessarily a problem, but if a developer expects to get the data using a `ref`, they won't get anything from the `ref.current.value` attribute.

I'd leave this as is until it does become a problem and we can figure out if we need to return data through the ref, somehow.